src/Serializer/Filter/GroupFilter.php

```diff
@@ -60,6 +64,32 @@ public function apply(Request $request, bool $normalization, array $attributes,
         $context[AbstractNormalizer::GROUPS] = $groups;
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function applyToSerializerContext(string $resourceClass, string $operationName, bool $normalization, array $context, array &$serializerContext): void
+    {
+        if (\array_key_exists($this->parameterName, $commonAttribute = ($context['request_attributes']['_api_filters'] ?? []))) {
```

Comment:
Filters should already be available in `$context`.

Reply:
I can check this :+1:

Reply:
Seems OK to do it. There is not related Behat tests though.